modules/ROOT/pages/fd-connection-switch-in-project.adoc

```diff
@@ -0,0 +1,52 @@
+= Switch the Connection That a Connector is Using in a Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+Every connector that you include in a flow in a Mule-app project is configured to use a connection to connect at runtime to the system that the connector is for. A _connection_ is a saved set of information that a running Mule app must use to connect to a specific system by means of a connector. For example, you configure a Salesforce connector to use a specific connection to Salesforce, a Database connector to use a specific connection to a database.
+
+You can change a configuration to use a different connection or you can create and use a configuration that uses a different connection.
+
+For example, suppose that you add a Database connector to a flow. When you create configuration Config 1 for it, you create Connection MySQL to specify the information that the your Mule app needs to connect to a MySQL database at runtime. Now, you want to test your Mule app against an Oracle database.
+
+Because the configuration for the Oracle database requires not only a separate connection, but also different additional information, you create the configuration Config 2 and the connection Oracle. Then, you switch to using Config 2.
+
+.For every configuration that you create for a connector, in the *Advanced* section you can specify information in addition to the connection for the configuration to use.
+image::fd-config-advanced.png[The Advanced section of the Database connector]
+<1> The text to click to open the *Advanced* section.
+<2> The additional information that you can specify for the Database connector.
+
+This diagram depicts the switch from Config 1 to Config 2:
+
+.This diagram shows that you can switch from one configuration to another to switch connections.
+image::fd-connections-configs-connectors-switch-1.png[Switching configurations to switch connections]
+
+In this next diagram, the Database connector is using the configuration Config 1, which uses connection MySQL1. If you want to switch to a different MySQL database and connecting to the database does not require any differences in the information that the configuration provides, you can simply change the configuration to use the connection MySQL2.
+
+.This diagram shows that you can switch connections in a configuration.
+image::fd-connections-configs-connectors-switch-2.png[Switching connections in a configuration]
+
+== Procedure
+
+* If you want to switch connections and the configuration requires different additional information, create a configuration that uses the connection you want to switch to.
++
+. Open the card that represents the connector that you need to switch to a different connection.
+. At the top of the card, click *Switch configurations*.
+. Click *Add*.
+. Create the new configuration, selecting or creating the connection that you want to use, and click *Save*.
```

Comment:
I suggest following the step above directly with this result, so that the final step is not lost under "Results." I suggested this in a previous topic as well; if I see this again, I will not mark it but leave it to you to decide which presentation you think is best.

"The *Connector configuration* area of the card shows that the new configuration is selected. Also, the configuration is listed in the *Reusable Configurations* section of the *Project* pane on the left side of the canvas.
. Click *Back*"